lib/user_agent/browsers/edge.rb

```diff
@@ -4,7 +4,7 @@ class Edge < Base
       OS_REGEXP = /Windows NT [\d\.]+|Windows Phone (OS )?[\d\.]+/
 
       def self.extend?(agent)
-        agent.last && agent.last.product =~ /Edge?/
+        agent.last && %w(Edge Edg EdgA EdgiOS).include?(agent.last.product)
```

Comment:
This is taken from https://github.com/art19/useragent/blob/35e58f45421e4dcc69dfde6a76735c740de495d6/lib/user_agent/browsers/edge.rb#L11